dmtest_main.cxx

```diff
@@ -0,0 +1,108 @@
+// #include "bout/mpi_wrapper.hxx"
```

Comment:
If this program is a test, should it be in a test directory. If it's a program a comment here about it does.